doc/flame_fire_atlas/fire_atlas.md

```diff
@@ -0,0 +1,86 @@
+# FireAtlas
+
+FireAtlas is a tool for handling texture atlases.
```

Comment:
It needs more information about what texture atlases are and how they are used, like how you map string keys to assets.

Reply:
updated